Add newmembers

fiestared opened the pull request:
@sot528 I added new members' profile. Please check it.

sot528 commented on src/yaml/en.yml:
```diff
@@ -143,21 +143,57 @@ member:
+      body: |
+        After graduating from Tokyo Denki University Faculty of Engineering in 2008, Yamashita joined a banking service development company where he was involved in customizing packages for regional banks, transitioning to a joint center operations. Interested in the development of open systems rather than the development of general purpose systems, in 2012, Yamashita changed to a web development ...
+      role: ENGINEER
+      twitter: https://github.com/yaasita
```
It looks GitHub's URL.
NOT Twitter's URL.

sot528 commented on src/yaml/ja.yml:
```diff
@@ -150,21 +150,57 @@ member:
+      body: |
+        東京電機大学工学部を2008年卒業後、銀行系サービス開発会社に入社。地方銀行向けパッケージのカスタマイズ、共同センター移行、運用に携わる。汎用系の開発よりオープン系の開発に興味を持ったため、2012年より、Web系開発会社に転職。LAMP環境の開発・運用に携わる。2017年よりフリーランス。ALISでは主に認証部分、検索部分の開発を担当。
+      role: ENGINEER
+      twitter: https://github.com/yaasita
```
It looks GitHub's URL.
NOT Twitter's URL.

sot528 commented on src/team.pug:
```diff
@@ -159,79 +175,168 @@ body
     member.profile.kamei.linkedin)
   .p-popup-overlay(data-exit=member.profile.kamei.id)
 
   article.p-popup.p-popup-member.is-effect(src='/img/pc/img_profile_usagi.png', id="js-popup-"+member.profile.usagi.id)
```
~~img_profile_usagi~~ => **img_profile_akagi**

sot528 commented:
Mrs. Kubori's first name is "CHISA". 
NOT "CHIKA"

Please fis it 🙏

sot528 commented:
I can not see member faces when select Japanese language.
Maybe it depends image file path.
<img width="1190" alt="2018-07-09 13 48 12" src="https://user-images.githubusercontent.com/2289647/42431165-db00c18a-837e-11e8-9d3d-cba4e5c3000b.png">

fiestared commented:
@sot528 I fixed all points you mentioned, but somehow I couldn't fix the Akagi's profile picture( I changed the profile path to /img/pc/img_profile_akagi.png , but still shown as a img_profile_usagi.png

sot528 commented:
@fiestared 
> I couldn't fix the Akagi's profile picture
Here you are ;)
- https://github.com/AlisProject/welcome/blob/master/src/mixin/popup_member.pug#L6

Actually, it it no good implementation...

sot528 commented:
I fixed it.
LGTM!
